src/backend/services/users-api/tests/functional/test_routes.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from unittest.mock import patch
+from src.databases import database_init
+import mongomock
+
+with patch.object(database_init, 'create_users_db', return_value=mongomock.MongoClient()):
+    from src.app import app
+    from src.app import mongo
+
+from src.databases.UserSchema import userSchema
+import sys, os
+
```

Comment:
Don't forget to update these tests when endpoint types and names of fields change as a result of the above comments!

Reply:
done